Description: Remove rubygems and bundler
 apt takes care of the required dependencies.
 we note down dependencies in debian/control.
Author: Praveen Arimbrathodiyil <praveen@debian.org>
Last-Updated: 2013-05-23

--- a/test/helper.rb
+++ b/test/helper.rb
@@ -1,4 +1,3 @@
-require "bundler/setup"
 require "simplecov"
 require "simplecov-html"
 require "test/unit"
--- a/lib/simplecov-html.rb
+++ b/lib/simplecov-html.rb
@@ -8,7 +8,7 @@ require "time"
 major, minor, patch =  SimpleCov::VERSION.scan(/\d+/).first(3).map(&:to_i)
 if major < 0 || minor < 9 || patch < 0
   fail "The version of SimpleCov you are using is too old. "\
-  "Please update with `gem install simplecov` or `bundle update simplecov`"
+  "Please update"
 end
 
 module SimpleCov
